technical/indicators/momentum.py

```diff
@@ -65,7 +65,7 @@ def ultimate_oscilator(dataframe):
 
 # WILLR                Williams' %R
 def williams_percent(dataframe, period=14, field='close'):
-    highest_high = dataframe[field].rolling(period).max()
-    lowest_low = dataframe[field].rolling(period).min()
+    highest_high = dataframe['high'].rolling(period).max()
+    lowest_low = dataframe['low'].rolling(period).min()
     wr = (highest_high - dataframe[field]) / (highest_high - lowest_low) * -100
```

Comment:
does the "field" parameter still make sense with this change? 
Williams-percent seems ... pretty well defined, and doesn't really allow using a different field, so i'd question if it makes sense to also remove the "field" parameter, and have it use close in all cases.

Reply:
I don't think that the "field" param makes sense. I checked the freqtrade-strategies repository to see if removing it would break any code, and it wouldn't. I'll make the fix in a few minutes.